.github/workflows/tests.yml

```diff
@@ -30,6 +30,35 @@ jobs:
           cd osbuild
           pylint osbuild runners/* assemblers/* stages/* sources/*
 
+  documentation:
+    name: "📚 Documentation"
+    runs-on: ubuntu-latest
+    container:
+      image: docker.io/library/python:3.7
+    steps:
+      - name: Install Dependencies
+        run: |
+          pip install docutils
+
+      - name: Clone repository
+        uses: actions/checkout@v2
+        with:
+          path: osbuild
+
+      - name: Generate Documentation
+        run: |
+          make \
+            -f osbuild/Makefile \
+            SRCDIR=osbuild \
+            BUILDDIR=build \
+            man
+
+      - name: Verify Documentation
+        working-directory: build
+        run: |
+          test -d docs
+          test -f docs/osbuild.1
```

Comment:
Should we check for all the man pages here?

Reply:
I am unsure how elaborate these should be. I definitely don't want to hard-code every man-page here, but I do want to have an explicit check here which does not do some scripting, but is obvious and direct.

I am open to suggestions. Something like `for M in docs/*.rst ; do test -f $M ; done` would work, but what does it really grant us? I don't know. Ideas welcome! Maybe even use `man` to validate a file is a correctly formatted TROFF file.

Reply:
Ah, fair enough. I was tripped up by the wording in the commitmessage. Checking for the existence of one hardcoded manpage makes sense.
Maybe also the correctness of all the ones we find, but that can be separate, I guess.